pkg/artifactory/provider.go

```diff
@@ -232,11 +232,11 @@ func checkArtifactoryLicense(client *resty.Client) error {
 	license := License{}
```

Comment:
Hi.

Our internal artifactory has HA License information.

Their response is here.
```
{
  "licenses" : [ {
    "type" : "Enterprise Plus",
    "validThrough" : "Mar 31, 2023",
    "licensedTo" : "****",
    "licenseHash" : "****",
    "nodeId" : "one",
    "nodeUrl" : "****",
    "expired" : false
  }, {
    "type" : "Enterprise Plus",
    "validThrough" : "Mar 31, 2023",
    "licensedTo" : "****",
    "licenseHash" : "****",
    "nodeId" : "two",
    "nodeUrl" : "****",
    "expired" : false
  }]
}
```

This does not match `License{}`, so checking will fail always...

Reply:
@cappyzawa Good catch!